src/resolveVsCodeGlobalVariables.ts

```diff
@@ -0,0 +1,103 @@
+import path from "path";
+import process from "process";
+import vscode from "vscode";
+
+// taken from https://github.com/DominicVonk/vscode-variables converted to ts and modified
```

Comment:
We can just add this as a dependency. The current copying of the code here violates the MIT license the package is released under.

Reply:
actually, that's problematic. Making a quick fork.

Reply:
@c4312/vscode-variables is published

Reply:
thx - using it now 🤗